src/pages/selectSalesOfferPackage.tsx

```diff
@@ -66,17 +65,24 @@ export interface SelectSalesOfferPackageProps {
     csrfToken: string;
 }
 
+export const formatSOPArray = (stringArray: string[]): string => {
+    let stringToReturn = '';
+    stringArray.forEach(string => {
+        stringToReturn = `${stringToReturn}${stringToReturn.length > 0 ? ',' : ''} ${startCase(string)}`;
+    });
+    return stringToReturn;
+};
+
 const generateCheckbox = (
     salesOfferPackagesList: SalesOfferPackage[],
     productName: string,
     selected?: { [key: string]: string[] },
 ): ReactElement[] => {
     return salesOfferPackagesList.map((offer, index) => {
-        const { name, description } = offer;
-        let checkboxTitles = `${name} - ${description}`;
-
-        if (checkboxTitles.length > 200) {
-            checkboxTitles = `${checkboxTitles.substr(0, checkboxTitles.length - 10)}...`;
+        const { name, description, purchaseLocations, paymentMethods, ticketFormats } = offer;
+        let modifiedDescription = description;
+        if (modifiedDescription.length > 200) {
```

Comment:
Since the description is restricted to 150 characters by the API we can probably remove this if statement entirely